### PR TITLE
Add GiTiny by @k-lpmg

### PR DESCRIPTION
**Pull request by k-lpmg**
1. [x] Project URL: https://github.com/k-lpmg/GiTiny
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17630,6 +17630,32 @@
...
+        "itunes": "https://itunes.apple.com/app/tiny-github/id1461424565",
+        "license": "mit",
+        "stars": 106,
+        "tags": [
+            "swift",
+            "rxswift",
+            "mvvm-c",
+            "clean-architecture"
+        ],
+        "screenshots": [
```
Comment (vpeschenkov) [inline contents.json]: @k-lpmg the screenshots links should be direct to images.

File: contents.json [hunk 2]
```diff
@@ -17630,6 +17630,32 @@
...
       "suggested_by": "@Karambirov",
       "stars": 1
+    },
+    {
+        "title": "GiTiny",
+        "category-ids": [
+            "github",
+            "rxswift"
+        ],
+        "description": "iOS app for GitHub with exploring trending",
```
Comment (vpeschenkov) [inline contents.json]: @k-lpmg could you remove "iOS" from the description? Might this be good instead "An app for GitHub with exploring trending"

**Comment by k-lpmg** [comment]
@vpeschenkov I updated what you requested.

**Comment by vpeschenkov** [comment]
@k-lpmg thank you!

**Pull request merged**